packages/suite/src/components/suite/AppNavigation/index.tsx

```diff
@@ -199,117 +280,130 @@ const AppNavigation = ({ items, primaryContent }: Props) => {
     const itemsSecondaryWithExtra = itemsSecondary.filter(item => item.extra);
     const itemsSecondaryWithoutExtra = itemsSecondary.filter(item => !item.extra);
 
+    const sticky = isInView();
+    const selectedAccount = useSelector(state => state.wallet.selectedAccount.account);
+
     return (
-        <Wrapper ref={wrapper} settingsWrapper={routeName && routeName.startsWith('settings')}>
+        <Wrapper ref={wrapper} value={sticky} subRoute={routeName && isSubsection(routeName)}>
             {routeName && isSubsection(routeName) ? (
-                <Primary>
-                    <StyledBackLink onClick={() => goto('wallet-index', undefined, true)}>
-                        <StyledIcon icon="ARROW_LEFT" size={16} />
-                        <Translation id="TR_BACK" />
-                    </StyledBackLink>
-                </Primary>
+                <InnerWrap>
+                    <MenuHolder
```

Comment:
MenuHolder component is used only here so `settingsWrapper` and `border` conditions could be removed completely from it, because they have always the same value here because of the enveloping condition.

Reply:
331a45e053a53457e46506ccff18f29dd5f5b5fc